README.md

```diff
@@ -1,5 +1,6 @@
 ## Default Bahmni configuration and data. 
 ======================================================================
+This repo also includes additional configurations required for   [FHIR CDSS module](https://github.com/Bahmni/openmrs-module-cdss) and [FHIR Terminology Service Module](https://github.com/Bahmni/openmrs-module-snomed) along with other modules bundled with standard Bahmni.
```

Comment:
Recommend putting the link to SNOMED_BAHMNI Wiki page. That will be more accurate, than listing these repos.